src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -50,6 +52,37 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * @private 
   */
   grantLocationAccess_() {
-    //ToDo(kristenmason@): implement map
+    //ToDo(kristenmason@): implement the Google Maps request access functionality
+    this.simidProtocol.sendMessage(CreativeMessage.REQUEST_PAUSE).then(() => {
+      findNearest.classList.add("hidden");
+      //ToDo(kristenmason@): display map
+      this.createButtons_();
+    })
+  }
+
+  createButtons_() {
+    const returnToAdButton = document.createElement("button");
+    returnToAdButton.innerHTML = "Return To Ad";
+    returnToAdButton.id = "returnToAd";
+    returnToAdButton.onclick = () => this.playAd_(returnToAdButton); 
```

Comment:
having a returnToAd button is super uncommon.  The more common thing to do is to just have a skip button only.

I wonder if what this is, is more of a "close map" button, perhaps it should be called that.

Reply:
The reason we designed it this way is if users wish to return to the ad to watch the rest of the video and learn more about the product. IMO "Return to Ad" is more informative than just close map.